lib/mcb.rb

```diff
@@ -411,6 +414,11 @@ def process_opt_changed_since(opts, url)
         changed_since_param = CGI.escape(changed_since.strftime('%FT%T.%6NZ'))
         new_url.query = "changed_since=#{changed_since_param}"
       end
+
+      if opts.key? :recruitment_year
+        new_url.query = [new_url.query, "recruitment_year=#{opts[:recruitment_year]}"].compact.join('&')
+      end
```

Comment:
How about generating a hash and using `to_query` to generate a query from it? Just thinking it'd be a tidier.

```
$ rails c
> { a: 'a', b: 'b' }.to_query
=> "a=a&b=b"
```